modules/ROOT/pages/_partials/sync/provisioning.adoc

```diff
@@ -0,0 +1,62 @@
+
+It's neccessary to wrap your Sync Server application into Docker container and publish your container to a Docker registry. 
+The easiest way is to use a public one such as link:https://hub.docker.com/[Dockerhub].
```

Comment:
Maybe we should be more direct in our language.
"Before we provision the Sync Server we will build a Docker formatted container, deploy it to a public registry."  (I also think there is some legalese around saying Docker container vs Docker formatted container)

Reply:
ok, but will stick with Docker container, it's even on the page https://www.docker.com/resources/what-container